src/main/java/com/sandvoxel/generitech/common/tileentities/power/TestPower.java

```diff
@@ -16,6 +16,7 @@
 import net.darkhax.tesla.api.ITeslaProducer;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'net.darkhax.tesla.api.ITeslaProducer'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.aadev.xyz/coding_rules#rule_key=squid%3AUselessImportCheck)